BuzzerWolf.WebFrontend/next.config.ts

```diff
@@ -0,0 +1,8 @@
+import type { NextConfig } from "next";
+
+const nextConfig: NextConfig = {
+    /* config options here */
+  output: 'export', // required for NextJS static .net hosting
```

Comment:
this line `output: 'export'` is very important for the aspnet static render magic. it's not a default setting.